packages/amplication-client/src/Resource/create-resource/wizard-pages/CreateGithubSync.tsx

```diff
@@ -14,7 +14,7 @@ import { Icon, ToggleField, Tooltip } from "@amplication/ui/design-system";
 import AuthWithGitProvider from "../../git/AuthWithGitProvider";
 
 const TOOLTIP_DIRECTION = "n";
-const CLASS_NAME = "create-git-sync";
+const CLASS_NAME = "create-git-sync1";
```

Comment:
why not using the same class ?

Reply:
> why not using the same class ?

Previously it was not same, i changed that.